google/cloud/bigtable/examples/data_async_snippets.cc

```diff
@@ -38,16 +38,15 @@ void PrintUsage(std::string const& cmd, std::string const& msg) {
 void AsyncApply(google::cloud::bigtable::Table table,
                 google::cloud::bigtable::CompletionQueue cq,
                 std::vector<std::string> argv) {
-  if (argv.size() != 3U) {
-    throw Usage{"async-apply: <project-id> <instance-id> <table-id> <row-key>"};
+  if (argv.size() != 2U) {
```

Comment:
Do we have a compiler flag enabled that makes us write `2U` here? Integer promotion rules say that the `2` integer literal should be promoted to the larger unsigned type safely.

Reply:
I am not aware of any. I think I may have introduced the extra `U` because in google `EXPECT_EQ(1, foo.size())` does produce warnings with GCC (without `-Wextra` even) (but I suspect there is a variable involved there).

In any case. I will merge this and then make a pass removing the `U`s.

Reply:
Oof. That sounds like a googletest bug. Maybe it was fixed, because it seems to not warn anymore (https://godbolt.org/z/Yyzq7D)